index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" type="text/css" href="./public/style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script type="text/javascript" src="bundle.js" defer="defer"></script>
+  <title>Chitter</title>
+</head>
+<body>
+  <div class='main_body' id='mainPage'>
+    <div id='leftColumn' class='column_left'>
```

Comment:
I like how you've divided it up into columns, it looks like it will have saved you and the user a lot of time when dipslaying peeps